### DISCUSSION TITLE
Refresh

### DISCUSSION THREAD

**Discussion by Tapanila** [microsoftgraph/msgraph-bicep-types#63] [General]
Any news on refresh? Would be especially interested on being able to run deployment with service principals so I could try this out with Azure DevOps pipelines

**Comment by dkershaw10** [comment]
Sorry for the delayed response.  We're hoping to have this available towards the end of this month. Will confirm nearer the time.

**Comment by dkershaw10** [comment]
In case anyone comes back to see this discussion, the deployment with service principals has been delayed until April.  Apologies for that -it's not what our team wanted either.  For more details on what we are working on and timelines please see https://github.com/microsoftgraph/msgraph-bicep-types/discussions/68#discussioncomment-8372829
